benchmarks/inference-server/text-generation-inference/manifest-templates/text-generation-inference.tftpl

```diff
@@ -0,0 +1,75 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: text-gen-infer
+  namespace: ${namespace}
+  labels:
+    app: text-gen-infer
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: text-gen-infer
```

Comment:
Is this common short name for TGI? Why not just `text-generation-inference`?

Reply:
not a common name, simply shorter so it's easier to type out the name when interacting via "kubectl". "tgi" is a common shorthand name we could also consider.

@achandrasekar if we change the name here, please also update the locust README where-ever the name is referenced.

Reply:
Updated all of them to tgi